t/139-ssl-cert-by.t

```diff
@@ -11,7 +11,7 @@ my $openssl_version = eval { `$NginxBinary -V 2>&1` };
 if ($openssl_version =~ m/built with OpenSSL (0|1\.0\.(?:0|1[^\d]|2[a-d]).*)/) {
     plan(skip_all => "too old OpenSSL, need 1.0.2e, was $1");
 } else {
-    plan tests => repeat_each() * (blocks() * 6 + 4);
+    plan tests => repeat_each() * (blocks() * 6 + 4) - 6;
```

Comment:
Need to avoid using `- num` in `plan`, otherwise the `ONLY` section will not work anymore.

Reply:
update